task_1.py

```diff
@@ -0,0 +1,37 @@
+phone_book = {}
+
+while True:
+    user_data = input("Enter data(use command :add,delete,stats,show,list):")
+    if user_data == "last":
+        break
+
+    data = user_data.split()
+    command = data[0]
+    name = data[1] if len(data) > 1 else None
+
+    if command == "add":
+        phone = data[2]
+        if name and name not in phone_book:
+            phone_book[name] = phone
+            try:
+                value = int(phone)
+            except ValueError:
+                print("The phone number can only contain numbers")
```

Comment:
nit: don't see the usage of `value`. I suppose you wanted to do something like:
```suggestion
            try:
                phone_book[name] = int(phone)
            except ValueError:
                print("The phone number can only contain numbers")
```

Reply:
i fixed it